sdk/scheduler/src/main/java/com/mesosphere/sdk/framework/ProcessExit.java

```diff
@@ -29,6 +33,8 @@ public static void exit(Code code) {
         String message = String.format("Process exiting immediately with code: %s[%d]", code, code.getValue());
         System.err.println(message);
         System.out.println(message);
+        System.err.println("Printing final thread state");
```

Comment:
nit: trailing dots to indicate action in progress to keep in line with other log statements